projects/Mallard/src/screens/settings/gdpr-consent-screen.tsx

```diff
@@ -125,7 +126,9 @@ const GdprConsent = ({
                         appearance={ButtonAppearance.skeleton}
                         onPress={() => onEnableAllAndContinue()}
                     >
-                        Enable all and continue
+                        {shouldShowDismissableHeader
```

Comment:
Reading this code for the first time `shouldShowDismissableHeader` doesn't look like it should be coupled to that text. Perhaps we could just have `continueText` as a prop? Otherwise if I've misunderstood then 👌

Reply:
Will change.